dev/bazel/toolchains/cc_toolchain_lnx.bzl

```diff
@@ -154,6 +155,12 @@ def _add_gcc_toolchain_if_needed(repo_ctx, cc):
     else:
         return []
 
+def _add_sycl_linkage(repo_ctx, cc):
+    if ("icx" in cc) or ("icpx" in cc):
+        return ["-fsycl"]
```

Comment:
Do we want to extend these flags with -fsycl-device-code-split=per_kernel, we use it in dpcpp.mk?

Reply:
This is currently only being used in order to get the include paths of the compiler when it uses SYCL. From some experiments, it looks like adding `-fsycl-device-code-split=per_kernel` on top of `-fsycl` doesn't add any new include paths.

The flag `-fsycl-device-code-split=per_kernel` is however added as a compiler argument later on:
https://github.com/oneapi-src/oneDAL/blob/488c4006a5a2f29c272c1c48e71b7a3a2ece5a61/dev/bazel/toolchains/cc_toolchain_lnx.bzl#L248